lib/coophub/backends/github.ex

```diff
@@ -89,20 +76,36 @@ defmodule Coophub.Backends.Github do
 
   @impl Backends.Behaviour
   @spec prepare_request_languages(org, repo) :: request_data
-  def prepare_request_languages(%Organization{key: key}, %Repository{name: name}) do
-    prepare_request("#{key}/#{name}", "repos/#{key}/#{name}/languages")
+  def prepare_request_languages(%Organization{login: login}, %Repository{name: name}) do
+    prepare_request("#{login}/#{name}", "repos/#{login}/#{name}/languages")
   end
 
   @impl Backends.Behaviour
   @spec parse_languages(langs) :: langs
   def parse_languages(languages) do
-    languages
+    Repos.get_percentages_by_language(languages)
+  end
+
+  @spec get_org_languages(org()) :: map
+  def get_org_languages(%Organization{:repos => repos}) do
```

Comment:
This one is exactly the same as the `gitlab` one! Why it was moved here?

Reply:
Moved to Repos!